plugin.cpp

```diff
@@ -300,7 +314,7 @@ PLUGIN_HANDLE plugin_init(ConfigCategory* config)
 		
 		if ( toAddress.size() != toNames.size() )
 		{
-			Logger::getLogger()->error("There is mistach between To address and To names count");
+			Logger::getLogger()->error("There is mistach between To address and To names count, exiting...");
```

Comment:
Do not saying citing in the logs, it doesn't/shouldn't exit as this will take down the notification service and is not what we want at all. Have you tested this scenario to know what actually happens?

Reply:
Yes, I tested it, It's just retuning null so no o email won't be sent out if user doesn't provide name or every email address.